components/common/IdentityForm.vue

```diff
@@ -122,13 +124,14 @@ const identity = ref<Record<string, string>>({
   legal: '',
 })
 const deposit = ref('0')
-const inputLengthLimit = ref(32)
+const inputLengthLimit = ref('32')
```

Comment:
to fix a bug where Basic input complain it expects string props

Reply:
or change `maxlength?: string` to `maxlength?: string | number`

Reply:
done